diff --git a/node_modules/@capacitor/app/CapacitorApp.podspec b/node_modules/@capacitor/app/CapacitorApp.podspec
index 2406e7f..3d5d5cb 100644
--- a/node_modules/@capacitor/app/CapacitorApp.podspec
+++ b/node_modules/@capacitor/app/CapacitorApp.podspec
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.author = package['author']
   s.source = { :git => 'https://github.com/ionic-team/capacitor-plugins.git', :tag => package['name'] + '@' + package['version'] }
   s.source_files = 'ios/Sources/**/*.{swift,h,m,c,cc,mm,cpp}', 'app/ios/Sources/**/*.{swift,h,m,c,cc,mm,cpp}'
-  s.ios.deployment_target  = '13.0'
+  s.ios.deployment_target  = '12.0'
   s.dependency 'Capacitor'
   s.swift_version = '5.1'
 end
